File: solutions/1662. Check If Two String Arrays are Equivalent.cpp
class Solution {
public:
    bool arrayStringsAreEqual(vector<string>& word1, vector<string>& word2) {
        string s = "", t = "";
        for (string x : word1)
            s += x;
        
        for (string x : word2)
            t += x;
        
        return s==t; 
    }
};

File: solutions/1662. Check If Two String Arrays are Equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        StringBuilder sb1 = new StringBuilder();
        for (String w : word1) {
            sb1.append(w);
        }
        
        StringBuilder sb2 = new StringBuilder();
        for (String w : word2) {
            sb2.append(w);
        }
        
        return sb1.toString().equals(sb2.toString());
    }
}
